### DISCUSSION TITLE
How many clients can be active at the same time?

### DISCUSSION THREAD

**Discussion by nevalashka** [wg-easy/wg-easy#497] [Q&A] [answered]
I have successfully running wg-easy on ubuntu 22.04 with 1 vCPU and 1 gb ram. How many clients approximately can be connected at once to this server?

**Comment by redsawah** [comment]
near 200
but what the bandwidth and cpu?

**Reply by nevalashka** [reply]
1 core Intel Xeon E3-12xx v2 2199.998 MHz and 1 gb/s and 1818 Mbit/s of download speed

**Reply by redsawah** [reply, marked as answer]
i think you can setup 200 clients
